custom_components/tuya_local/devices/dynasty_btx_fireplace.yaml

```diff
@@ -0,0 +1,193 @@
+name: Fireplace
+products:
+  - id: 5081205010521cfac95f
+    name: Dynasty BTX Fireplace
+primary_entity:
+  entity: climate
+  icon: "mdi:fireplace"
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          value: heat
+    - id: 3
+      name: current_temperature
+      type: integer
+      range:
+        min: 0
+        max: 40
+    - id: 5
+      name: temperature
+      type: string
+      mapping:
+        - dps_val: "0"
+          value: 16
+        - dps_val: "1"
+          value: 19
+        - dps_val: "2"
+          value: 21
+        - dps_val: "3"
+          value: 23
+        - dps_val: "4"
+          value: 25
+        - dps_val: "5"
+          value: 27
+        - dps_val: "6"
+          value: 30
+    - id: 13
+      name: temperature_unit
```

Comment:
Including this here will require more complex handling of temperature dp, as you need the values and range to be different depending on this setting.
It may be better to leave this out, and mark this config as only being valid for C.